components/pages/create-deliverable/view.tsx

```diff
@@ -91,10 +114,27 @@ export default function CreateDeliverablePageView({
                 <BountyLabel className="mb-2 text-white">
                   {t("deliverable:create.labels.preview")}
                 </BountyLabel>
-                <div className="d-flex justify-content-center border-dotter border-radius-4 border-gray-800">
-                  <span className="p-5 text-gray-800">
-                    {t("deliverable:create.placeholders.preview")}
-                  </span>
+
+                <div className="d-flex justify-content-center border border-radius-4 border-gray-800 comment">
+                  {previewIsLoading ? (
+                    <span className="spinner-border spinner-border m-5" />
+                  ) : (
+                    <>
+                      {previewLink?.ogImage && (
+                        <img src={previewLink.ogImage} />
+                      )}
+                      {previewLink?.ogVideo && !previewLink?.ogImage && (
+                        <video src={previewLink.ogVideo} controls>
+                          {t("deliverable:actions.preview.video-error")}
+                        </video>
+                      )}
+                      {!previewLink?.ogImage && !previewLink?.ogVideo && (
+                        <span className="p-5 text-gray-800">
+                          {t("deliverable:create.placeholders.preview")}
```

Comment:
switch to "no preview available for <url>"

Reply:
In the design it keeps the default name but gives an error in the input that a problem occurred in the load